src/Components/List/List.jsx

```diff
@@ -78,7 +78,11 @@ export default function List({ items, token }) {
               aria-controls="itemsList"
             />
             {searchItem !== '' && (
-              <button className="resetButton" onClick={resetSearch}>
+              <button
+                aria-label="Reset"
```

Comment:
Thanks for adding the `aria-label`! This is non-blocking, but consider adding a more descriptive `aria-label` like "Reset search form to show all list items". I feel that "Reset" is still a little mysterious especially if you're visually impaired.

Reply:
I like this suggestion, @alexandra-lim! I think we can split the difference here. Since this text input is understood as a search area, we could label this button as `Reset search`, or perhaps `Clear search`.

Reply:
Thanks for pointing that out @alexandra-lim and thanks for the suggestions @mxmason ! We changed the label to `Clear search`.